blocks/library/cover-image/style.scss

```diff
@@ -10,6 +10,9 @@
 			color: white;
 			font-size: 24pt;
 			line-height: 1em;
+			max-width: $visual-editor-max-width;
+			padding: $block-padding;
+			text-align: center;
```

Comment:
I guess some of these are `editor` specific while the `text-align` might be shared with the frontend styling too.
This is just a note to take these considerations into account while rebasing.